lib/events/codes.go

```diff
@@ -542,6 +542,11 @@ const (
 	// SecReportsReportRunCode is used when a report in run.
 	SecReportsReportRunCode = "SRE002I"
 
+	// ExternalAuditStorageEnableCode is the External Audit Storage enabled code.
+	ExternalAuditStorageEnableCode = "TEAS001I"
+	// ExternalAuditStorageDisableCode is the External Audit Storage disabled code.
+	ExternalAuditStorageDisableCode = "TEAS002I"
```

Comment:
Is there any rhyme or reason to the naming of these codes besides

1. unique per semantic
2. suffix with: `I` = info, `W` = warning, `E` = error

I like `"TEAS"` although I will note that it's breaking precedent with a 4 letter prefix, afaik the max thus far is 3